commands/logs_e2e_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Display single log line", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+	})
+
+	s.RunForAllClients("Display in logrus for formatting", func(c client.Client) {
```

Comment:
Hence, this should be `RunForSystemAdminAndLocal`

Reply:
updated